ts/torch_handler/base_handler.py

```diff
@@ -101,7 +118,14 @@ def inference(self, data, *args, **kwargs):
         """
         marshalled_data = data.to(self.device)
         with torch.no_grad():
-            results = self.model(marshalled_data, *args, **kwargs)
+            if self.torch_api_type == 'python':
+                results = self.model(marshalled_data, *args, **kwargs)
+            else:
+                if len(kwargs):
+                    raise Exception("Keyword arguments are not supported by CPP API. Use Python API instead.")
```

Comment:
What are the kwargs used for at present for python side? Can you please clarify, what aspects will not work with C++ due to this with an example?